### ISSUE TITLE
Write documentation for maintenance tools

### ISSUE THREAD

**Issue by rfrenchseti** [SETI/rms-pdsfile#45]
The "maintenance" (previously "validation") tools are mostly undocumented. We need to write user-level documentation describing their use, and also improve internal documentation as needed.

**Comment by mseritan** [comment]
I am writing a document called `holdings-update-instructions.md` that will eventually live in `rms-webserver`. It covers how to download new datasets, how to arrange them in the (PDS3) holdings, the scripts to run to create all the auxiliary files, and how to check dependencies/validate, etc. This mostly falls under "improve internal documentation" mentioned in the issue description.

**Comment by mseritan** [comment]
I've been adding to my PDS3 holdings update instructions (`holdings-update-instructions.md` on the `holdings-doc-updates` branch of the rms-webserver repo) as I've been adding various PDS3 datasets to our holdings. The document has reached a state where it is ready to be fully reviewed so that it can be merged into main. The pull request for this is linked to this issue ([rms-webserver #133](https://github.com/SETI/rms-webserver/pull/133)). I've assigned @markshowalter as reviewer, though it could be someone else if Mark is working on other things. I am envisioning that once a review is complete and the document is merged into main, then this issue can finally be closed. @matthewtiscareno, what do you think?

**Comment by matthewtiscareno** [comment]
[Here](https://github.com/SETI/rms-webserver/blob/holdings-docs-updates/holdings-update-instructions.md) you can see the document properly formatted.
